tests/questions.test.js

```diff
@@ -0,0 +1,57 @@
+const axios = require('axios');
+
+const USERS_SERVICE_HOST = "http://peerpreptest.bryanlohxz.com/api/users-service";
+const QUESTIOS_SERVICE_HOST = "http://peerpreptest.bryanlohxz.com/api/questions-service"
```

Comment:
Store in environment variable so that we can use this on local services as well.